R/wcvp_match_names.R

```diff
@@ -149,7 +149,8 @@ wcvp_match_names <- function(names_df, wcvp_names = NULL, name_col = NULL, id_co
   if (fuzzy & nrow(unmatched) > 0) {
     cli_h2("Fuzzy matching {length(unique(unmatched[[name_col]]))} name{?s}")
     fuzzy_matches <- wcvp_match_fuzzy(unmatched, wcvp_names, name_col = name_col, progress_bar = progress_bar)
-
+    # non needed after fixing bug in edit_match fn?
+    #fuzzy_matches[is.na(fuzzy_matches$wcvp_name),"match_type"] <- "No match found"
```

Comment:
I would stick to your guns here and just delete this - it's in the commit history, and doesn't seem to break any tests